src/modules/hash.js

```diff
@@ -38,10 +38,13 @@ export class Hash {
         }
 
         this.lastHash = location.hash;
+ 
```

Comment:
Linter is now complaining of trailing white-spaces:
https://travis-ci.org/koalyptus/TableFilter/builds/144860663#L366